<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能家居-使徒行者</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>注册用户</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="UserNamelabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserNamelineEdit">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="PasswdLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密    码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswdLineEdit">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="AffirmLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确    认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AffirmLineEdit">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="PhoneLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>手    机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PhoneLineEdit">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="OKPushButton">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelPushButton">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
